File: Problemset/length-of-last-word/length-of-last-word.js

// @Title: 最后一个单词的长度 (Length of Last Word)
// @Author: 846188037@qq.com
// @Date: 2020-07-09 20:17:35
// @Runtime: 60 ms
// @Memory: 31.8 MB

/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    try{
    var list=s.split(" ")
    var i=1
    while (list[list.length-i].length==0)
    {
        i++
    }
    return list[list.length-i].length}
    catch(error){return 0}
};

File: Problemset/length-of-last-word/length-of-last-word.py

# @Title: 最后一个单词的长度 (Length of Last Word)
# @Author: 846188037@qq.com
# @Date: 2020-07-09 20:04:53
# @Runtime: 32 ms
# @Memory: 13.3 MB

class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        try:
            return len(s.split()[-1])
        except:
            return 0
